src/test/java/org/openstreetmap/atlas/geography/atlas/change/ChangeAtlasTest.java

```diff
@@ -50,8 +50,9 @@
     public void testBounds()
     {
         final Atlas atlas = this.rule.getAtlas();
-        Assert.assertEquals("POLYGON ((-122.2450237 37.5920679, -122.2450237 37.5938783, "
-                + "-122.2412753 37.5938783, -122.2412753 37.5920679, -122.2450237 37.5920679))",
+        Assert.assertEquals(
```

Comment:
Spotless?

Reply:
I ran `./gradlew spotlessApply` and this is the formatting it does. Maybe your `spotlessApply` is different from mine?